content/concepts/content-routing/overview.md

```diff
@@ -4,4 +4,56 @@ description: "Learn about Noise in libp2p."
 weight: 218
 ---
 
-Coming soon!
+## Overview
+
+Content routing refers to the process of directing data to its intended recipient within a
```

Comment:
Suggeston:

Content routing refers to the process of figuring out who has the data you want, and how to connect to them.